hadoop-hdds/docs/content/interface/Cli.md

```diff
@@ -208,6 +246,13 @@ $ ozone sh key info /vol1/bucket1/README.md
 $ ozone sh key get /vol1/bucket1/README.md /tmp/
 ```
 
+In delete key operation, if trash is enabled then for FSO bucket deleted key will move inside trash. 
+Whereas for OBS bucket keys will be deleted permanently irrespective of trash is enabled or disabled.
```

Comment:
```suggestion
If the key is in an FSO bucket it will be moved to the trash when deleted. 
If the key is in an OBS bucket it will be permanently deleted.
```

Probably want to link to the bucket types page for FSO/OBS buckets too, since I think that part is later in the docs and new users may not be familiar with this term. Also adding the default location of the trash directory will help.